buildpack.md

```diff
@@ -991,3 +993,19 @@ The stack ID:
 A buildpack descriptor that specifies `order` MUST be [resolvable](#order-resolution) into an ordering of buildpacks that implement the [Buildpack Interface](#buildpack-interface).
 
 A buildpack reference inside of a `group` MUST contain an `id` and `version`.
+
+### io.buildpacks.app.source OCI Image label
+The value of this label:
+- MUST be a string of escaped json complying with [RFC 8259](https://tools.ietf.org/html/rfc8259).
+- when unescaped, MUST comply with the following schema:
+```json
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "type": {"type": "string"},
```

Comment:
Based on the RFC there where two types explicitly defined. Should they be part of the schema or are they only guidance/examples of what it would look like. If the former, then we should be able to specify them using [`oneOf`](https://json-schema.org/understanding-json-schema/reference/combining.html#oneof) in json schema.

Reply:
These were examples, as there are potentially unlimited numbers of source code control systems (mercurial, bazaar, sourcesafe etc..).
We could specify `oneOf`, but only if we were all happy with limiting the supported source code control systems.